pvlib/spectrum/mismatch.py

```diff
@@ -571,3 +580,181 @@ def spectral_factor_caballero(precipitable_water, airmass_absolute, aod500,
     )
     modifier = f_AM + f_AOD + f_PW  # Eq 5
     return modifier
+
+
+def spectral_responsivity_to_quantum_efficiency(sr, wavelength=None):
```

Comment:
These function names are pretty long.  I would not be surprised if these would be the longest names in pvlib!  A minor problem is that they don't display perfectly in the documentation:

![image](https://github.com/pvlib/pvlib-python/assets/57452607/68cc6e3f-c3d2-4ddf-b993-53213fcb85e0)

Is `sr_to_qe` too short?  Or perhaps `calc_spectral_response(qe)` and `calc_quantum_efficiency(sr)`?

Reply:
Damm, good point. I prefer the shortest one, since it also states what the input data must be. But I don't have a strong opinion on it.